sympy/integrals/integrals.py

```diff
@@ -173,8 +176,8 @@ def transform(self, x, u):
         where `F(x)` is the inverse of `f(x)` and the limits and integrand have
         been corrected so as to retain the same value after integration.
 
-        Notes
-        =====
+        Explanation
+        ===========
```

Comment:
I think the above part is the explanation. I would leave this header here. We can't support subheaders yet https://github.com/sympy/sympy/issues/17618, but this will be one once we do.

Reply:
I think that this is the only comment to be addressed before merge here.

Reply:
Yes. Done.